modules/java/pages/_partials/gs-additional-steps-for-linux.adoc

```diff
@@ -29,16 +31,22 @@ NOTE: If the setenv.sh file doesn’t exist, you will need to create a new file.
 Desktop::
 +
 --
-. Copy the _native libraries_ (`<pathToCbl>/support`) to a location accessible to -- and usable by -- your build and runtime environments.
-. Add the following libraries to the `LD_LIBRARY_PATH` in the .bashrc file:
-
+. Within the `.zip` file, locate the support files appropriate for your environment.
```

Comment:
I think this sentence is redundant (here and in the previous section).  Immediately below, you recommend copying the entire support directory.  That is simple and will work fine.  But if you do that, there is no need to locate anything.